File: ch1-just-enough-ruby/p5-objects-and-methods.py
#!ipy
class MyObject(object):
    def add(self, x, y):
        return x + y
    def add_twice(self, x, y):
        return self.add(x, y) + self.add(x, y)

o = MyObject()
o.add(2, 3)
o.add_twice(2, 3)

File: ch1-just-enough-ruby/p5-objects-and-methods.rb
#!/usr/bin/irb --sample-book-mode
o = Object.new
def o.add(x, y)
    x + y
end
o.add(2, 3)
def o.add_twice(x, y)
    add(x, y) + add(x, y)
end
o.add_twice(2, 3)
